wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
This works fine for me.

When wifiphisher quits on success, this is the output:

[+] Captured credentials: wfphshr-wpa-password=supersecret&wfphshr-essid=Netgear&wfphshr-checkbox=checked&wfphshr-inputSelect=selection

Works with as many elements as you like.